test/javascripts/acceptance/composer-test.js.es6

```diff
@@ -101,81 +101,98 @@ QUnit.test("Tests the Composer controls", async assert => {
   assert.ok(!exists(".bootbox.modal"), "the confirmation can be cancelled");
 });
 
-// Temporarily remove to see if this is breaking the test suite
```

Comment:
7 months later... 😁
@SamSaffron `QUnit.skip` would have been the better choice. 😉

Reply:
:)